ios/LayoutReanimation/REAAnimationsManager.m

```diff
@@ -171,14 +182,15 @@ - (double)getDoubleOrZero:(NSNumber *)number
 
 - (void)setNewProps:(NSMutableDictionary *)newProps forView:(UIView *)view
 {
-  NSMutableDictionary *dataComponenetsByName = [_uiManager valueForKey:@"_componentDataByName"];
-  RCTComponentData *componentData = dataComponenetsByName[@"RCTView"];
-  [self setNewProps:newProps forView:view withComponentData:componentData];
+  NSMutableDictionary *dataComponentsByName = [_uiManager valueForKey:@"_componentDataByName"];
```

Comment:
Could we just do it once in `setNewProps` instead of doing it twice here and in `progressLayoutAnimationWithStyle`?

Reply:
Resolved by: https://github.com/software-mansion/react-native-reanimated/pull/4014/commits/dd59309b559dd81f0f5d4853fd675cc1d1dc99b2